src/crypto/store/indexeddb-crypto-store-backend.js

```diff
@@ -324,6 +328,77 @@ export class Backend {
         objectStore.put({deviceKey, sessionId, session});
     }
 
+    // Inbound group sessions
+
+    countEndToEndInboundGroupSessions(txn, func) {
```

Comment:
I think countEndToEndInboundGroupSessions is dead, and I'd be in favour of removing the impls.